### PR TITLE
Add info endpoint test

### PR DESCRIPTION
**Pull request by andrewmil**
Added a test to exercise the info endpoint using git_info file

### CODE REVIEW

File: git_info [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "test": "test"
+}
```
Comment (tej99) [inline git_info]: What is this file for?

File: tests/app/test_info.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+class TestInfo(unittest.TestCase):
+
+    def setUp(self):
+        self.app = app.test_client()
+
+    def test_info_with_git_info(self):
+        response = self.app.get("/info")
+
+        self.assertEqual(response.status_code, 200)
+        self.assertTrue('"name": "ras-frontstage"'.encode() in response.data)
```
Comment (gemmai95) [inline tests/app/test_info.py]: Could you not use assert in rather than use assert true in?

**Comment by rhiGriff** [comment]
Should the same test be added into the behave tests to test the info endpoint?

File: git_info (hunk 1 above)
Comment (andrewmil) [reply git_info]: :+1: this is a good idea

File: tests/app/test_info.py (hunk 2 above)
Comment (andrewmil) [reply tests/app/test_info.py]: This makes a lot of sense, I think we use this assertTrue method in pretty much all the tests in frontstage/frontstage-api but I'll have a look at changing them

**Comment by andrewmil** [comment]
@rhiGriff Probably, I haven't looked at the behave tests in a long time and wasn't aware we were running them anywhere currently. Are we going to use them for CI tests in the future?

File: git_info (hunk 1 above)
Comment (andrewmil) [reply git_info]: Though frontstage-api also uses this method and the file does get overwritten when it is deployed

**Comment by rhiGriff** [comment]
Behave tests have been used but it's not certain what will be used, although it is worth keeping the behave tests up to date

File: git_info (hunk 1 above)
Comment (tej99) [reply git_info]: Fair play, just think it's better to not have a default file for this. Similar to when we have env vars set to default values that can easily make their way to production.

Comment (tej99) [reply git_info]: https://github.com/ONSdigital/ras-frontstage-api/pull/22added the change to frontstage-api

**Pull request merged**